west.yml

```diff
@@ -137,7 +137,7 @@ manifest:
       groups:
         - fs
     - name: hal_adi
-      revision: de5dadb5322c5da9647363a0d081aa002d4b796f
+      revision: pull/9/head
```

Comment:
zephyrproject-rtos/hal_adi#9 has now been merged, so this can be updated to the new SHA